silkworm/db/freezer.cpp

```diff
@@ -156,6 +156,11 @@ Task<void> Freezer::cleanup() {
     if (range.first >= range.second) co_return;
     log::Debug(name()) << "cleanup [" << range.first << ", " << range.second << ")";
 
+    if (keep_blocks_) {
+        log::Debug(name()) << " skipping cleanup [" << range.first << ", " << range.second << ")";
```

Comment:
```suggestion
        log::Debug(name()) << "skipping cleanup";
```
The range is already printed above. The initial space is not needed, everything that is pushed to the logger is right-aligned as a separate column.